public/source/main.js

```diff
@@ -106,6 +106,7 @@ ko.bindingHandlers.autocomplete = {
 
 // Used to catch when a user was tabbed away and re-visits the page.
 // If fs.watch worked better on Windows (i.e. on subdirectories) we wouldn't need this
+// TODO is this still needed?
```

Comment:
I have checked it on windows and it seems that this is not needed (as the comment already states), but I will check linux and macOS as well.

Reply:
Well, node-watch falls back to fs.watch if it can recurse, so if fs.watch was fixed for Windows it can go I think

Reply:
Yes, I have checked the other os as well, so lets remove that.
Could you also update the [changelog](https://github.com/FredrikNoren/ungit/blob/master/CHANGELOG.md) please?

Reply:
done